usr/src/uts/common/brand/lx/syscall/lx_prctl.c

```diff
@@ -278,6 +278,16 @@ lx_prctl(int opt, uintptr_t data)
 		return (0);
 	}
 
+	case LX_PR_SET_NO_NEW_PRIVS: {
+		/*
+		 * On recent versions of Linux more services are starting to set
+		 * NoNewPrivs=yes in their systemd unit file. Since we currently
+		 * just return success for LX_PR_CAPBSET_DROP there is currently
+		 * no need to map this to the illumos privileges.
+		 */
```

Comment:
We should perhaps add @citrus-it 's comment in here as well:

"The problem is, AIUI, the Linux implementation is all at the thread level, whereas illumos privileges apply to the entire process."

Otherwise, @citrus-it is yanking it in to OmniOS, so perhaps we should as well?